docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -0,0 +1,300 @@
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` is truthy.
+
+```js
+await t.expect('ok').ok('this assertion will be passed');
+await t.expect(false).ok('this assertion will be failed');
+```
+
+## notOk
+
+```text
+await t.expect(actual).notOk(message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | Value that should be falsy
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` is falsy.
+
+```js
+await t.expect('ok').notOk('this assertion will be failed');
+await t.expect(false).notOk('this assertion will be passed');
+```
+
+## contains
+
+```text
+await t.expect(actual).contains(expected, message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | String or Array that should contains `expected` value
+`expected`             | Any type | Expected value
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` contains `expected`. `Actual` should be String or Array.
+
+```js
+await t.expect('foo bar').contains('bar', 'actual string contains expected');
+await t.expect([1, 2, 3]).contains(2, 'actual array contains expected');
+```
+
+## notContains
+
+```text
+await t.expect(actual).notContains(expected, message, options);
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | String or Array that should not contains `expected` value
+`expected`             | Any type | Expected value
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](index.md#assertion-options).
+
+Asserts that `actual` does not includes `expected`. `Actual` should be string or array.
```

Comment:
Asserts that `actual` does not include `expected`. `Actual` should be String or Array.